### DISCUSSION TITLE
Archive of old versions

### DISCUSSION THREAD

**Discussion by lordpengwin** [gravitational/teleport#43909] [Q&A]
Are old versions still available for download somewhere? Specifically I'm looking for teleport-v2.6.1-linux-amd64-bin.tar.gz

**Comment by webvictim** [comment]
I don't believe that version is still available anywhere.

**Comment by samouds** [comment]
Same here I am looking for version https://get.gravitational.com/teleport-v3.2.4-linux-arm-bin.tar.gz   archive but can't find it.

**Reply by webvictim** [reply]
Same deal, that version is very old and no longer available.

**Reply by samouds** [reply]
Is there any way to get it from your team or build it?

**Reply by samouds** [reply]
It was available the 1st july 2024 I think as I installed it

**Reply by webvictim** [reply]
You should be able to check out the v3.2.4 tag with Git and run `make release`

**Reply by samouds** [reply]
If we can find the archive as it was available 10 days ago so we can host it on our servers it will be great.

**Reply by webvictim** [reply]
Why do you want such an old version?

**Reply by samouds** [reply]
Because we have hundreds of devices using it already, and we can't upgrade to a newer version without testing and deploying it which will take time.
